scripts/predocs-build.js

```diff
@@ -0,0 +1,3 @@
+import { copyFileSync } from 'fs';
+
+copyFileSync('readme.md', 'index.md');
```

Comment:
Because vitest's entry file needs to be named `index.md`.
I also included this in the gitignore file to prevent having to update both locations.